### PR TITLE
feat: cart getters implementation

### PR DESCRIPTION
**Pull request by abocsan-plenty**

### CODE REVIEW

File: apps/web/components/CartPageContent/CartPageContent.vue [hunk 1]
```diff
@@ -1,18 +1,18 @@
 <template>
-  <div v-if="cart?.lineItems.length" class="md:grid md:grid-cols-12 md:gap-x-6" data-testid="cart-page-content">
+  <div v-if="cart?.items?.length > 0" class="md:grid md:grid-cols-12 md:gap-x-6" data-testid="cart-page-content">
```
Comment (maxiroellplenty) [inline apps/web/components/CartPageContent/CartPageContent.vue:2]:
```suggestion
  <div v-if="cart?.items?.length ?? 0 > 0" class="md:grid md:grid-cols-12 md:gap-x-6" data-testid="cart-page-content">
```

File: apps/web/components/CartPageContent/CartPageContent.vue [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
-          :image-url="image?.url"
-          :image-alt="image?.alt"
-          :name="name ?? ''"
-          :price="totalPrice?.amount || 0"
-          :special-price="unitPrice?.value?.amount || 0"
+          :attributes="cartGetters.getItemAttributes(cartItem)"
+          :image-url="cartGetters.getItemImage(cartItem)"
+          :image-alt="cartGetters.getItemImage(cartItem)"
+          :name="cartGetters.getItemName(cartItem) ?? ''"
+          :price="cartGetters.getItemPrice(cartItem).regular || 0"
```
Comment (maxiroellplenty) [inline apps/web/components/CartPageContent/CartPageContent.vue:10]:
```suggestion
          :price="cartGetters.getItemPrice(cartItem)?.regular || 0"
```

File: apps/web/components/CartPageContent/CartPageContent.vue [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
+          :special-price="cartGetters.getItemPrice(cartItem).special || 0"
```
Comment (maxiroellplenty) [inline apps/web/components/CartPageContent/CartPageContent.vue:11]:
```suggestion
          :special-price="cartGetters.getItemPrice(cartItem)?.special || 0"
```

**Review by maxiroellplenty — CHANGES_REQUESTED** [review]
fix possible null errors

File: apps/web/composables/useContent/index.ts [hunk 4]
```diff
@@ -1 +1 @@
-export * from './useContent';
+// export * from './useContent';
```
Comment (maxiroellplenty) [inline apps/web/composables/useContent/index.ts:1]: empty files are not allowed. So I guess we have to remove the index file or just comment the sdk section in the composable

Comment (abocsan-plenty) [reply apps/web/composables/useContent/index.ts]: removed files

**Review by maxiroellplenty — APPROVED** [review]

**Pull request merged**